### DISCUSSION TITLE
does skywalking 9.2.0 support springcloud-gateway 3.x

### DISCUSSION THREAD

**Discussion by john1337** [apache/skywalking#9613] [General]
does skywalking 9.2.0 support springcloud-gateway 3.x，i am using skywlking to trace gateway,but it didn't work

**Comment by wu-sheng** [comment]
You could find the 3.x tests here, https://github.com/apache/skywalking-java/blob/main/test/plugin/scenarios/gateway-3.x-scenario/support-version.list

And 9.2.0 is a server release, it doesn't have anything relative to Gateway support. Gateway of Spring is only a Java agent plugin, a small thing only.

**Reply by john1337** [reply]
got it,i am using 3.0.6 version which is not supported now

**Reply by wu-sheng** [reply]
Don't know, we don't test all. Who submitted the lateat upgrade did.
